src/ExpressionEvaluator/Core/Source/ResultProvider/Helpers/DynamicFlagsMap.cs

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Diagnostics;
+using Type = Microsoft.VisualStudio.Debugger.Metadata.Type;
+
+namespace Microsoft.CodeAnalysis.ExpressionEvaluator
+{
+    internal sealed class DynamicFlagsMap
+    {
+        private static readonly DynamicFlagsMap Empty = new DynamicFlagsMap(null, null, null);
+
+        private readonly Type _typeDefinition;
+        private readonly bool[] _dynamicFlagsArray;
+        private readonly int[] _startIndices;
```

Comment:
Assuming you're referring to "_dynamicFlagsArray", the suffix is there to clarify that it's not an instance of DynamicFlags.